### PR TITLE
JS week 1 homework

### PR DESCRIPTION
**Pull request by AlfiYusrina**
Now you can: 
- [x] change an image  by adding URL

- [x] add to do list 

@segersrobbert

### CODE REVIEW

**Review by bitsagarob — APPROVED** [review]
Great job! 
I'ts good that you kept all the comments, keeps it very clear.
The original homework will be a better challenge for you ;)

**Comment by AlfiYusrina** [comment]
@segersrobbert 
I've created one page of my booklist:
- [x] making an object to store my data
- [x] using for..in for iterating my obj
- [x] adding a bit of CSS (not yet responsive though)

**Comment by AlfiYusrina** [comment]
Update: Adding some comments.

**Review by bitsagarob — APPROVED** [review]
This might be the best pull requests I have ever received coaching Hack Your Future!
Amazing!

File: Week1/homework/app.js [hunk 1]
```diff
@@ -1,11 +1,197 @@
 'use strict';
 
-{
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
+  // This is an array that stores my books' information.
+  // Each element is an object that contains details.
+  const books = [
```
Comment (bitsagarob) [inline Week1/homework/app.js]: This is supposed to be an object. You could use Object.keys() to convert it to an object on the fly and then apply .sort() array method

Comment (AlfiYusrina) [reply Week1/homework/app.js]: I was using an object, then I wasn't sure how to use .sort() from an object, that was why I changed it directly to an array. But I've just fixed it with Object.keys(). So now it should be an object :) Thank's for the hint.

File: Week1/homework/app.js [hunk 2]
```diff
@@ -83,6 +83,91 @@
...
+    id: 'the_girl_who_takes_an_eye_for_an_eye',
+    title: 'The Girl Who Takes an Eye for an Eye',
+    author: 'David Lagercrantz',
+    status: 'Read',
+    review: 'https://www.goodreads.com/review/show/2272703977',
+    rating: 4
+  }
+};
+
+const booksArray = Object.keys(booksObj).map(key => booksObj[key]);
```
Comment (AlfiYusrina) [inline Week1/homework/app.js]: @segersrobbert Object.keys() is now added.

**Pull request closed without merging**